public/js/questionLock.js

```diff
@@ -0,0 +1,48 @@
+var waitTime;
+
+$(function () {
+    //hide submit button since question is locked
+    $(':submit').hide();
+    // Start Clock
+    initializeClock('clockdiv', new Date(Date.parse(new Date()) +  parseInt(waitTime)))
+});
+
+function getTimeRemaining(endtime) {
+    var t = Date.parse(endtime) - Date.parse(new Date());
```

Comment:
Why do we need to make these variables? Why don't we return right away?

Reply:
sure